src/main/webapp/repo-bootstrap/global/configuration/system-permission-mappings-config.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sorry I don't understand the point of this file if we already have global permissions file where the permissions for the `system_admin` are defined.

Reply:
Same with this one, needs to be removed

Reply:
K, will wait for the PR